tools/alpine/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM node:10-alpine
```

Comment:
For the directory structure, instead of having tools/alpine and tools/linux and then tools/build.sh it might be better to have just tools/build directory with files named similar to how they are named in system-test - Dockerfile.linux, Dockerfile.node10-alpine and so on. Since you'll need all of those files to build for each version of node anyway. And having files in the same directory by their purpose seems more reasonable they slicing them by environment.